### PR TITLE
added week homewrk files

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: react/week3/react_week3/src/FetchedData.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
+import React,{useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import Timer from './Timer.js';
+
+
+ const toDo_url = 'https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw';
```
Comment (obelitz) [inline react/week3/react_week3/src/FetchedData.js:6]: Values that never change - `constants` - are usually written in uppercase: `TODO_URL`

File: react/week3/react_week3/src/FetchedData.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    
+    return(
+        <div>
+            <h1>Todolist</h1>
+            <InputData addTodo={addTodo}/>
+            {}
+            <Timer />
+           { todo.length>0?
+           todo.map(todoItem=>(
+               < Children >
```
Comment (obelitz) [inline react/week3/react_week3/src/FetchedData.js:156]: You don't need a `Children` component here - you can simply add the div with the class right here:
```
<div className="styles">
  <Todo 
    ...
  />
</div>
```

File: react/week3/react_week3/src/FetchedData.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+}
+const Children =({children})=>{
+    return(
+        <div className='styles'>
+          {children}
+        </div>
+    )
+}
+
+function FetchedData(){
```
Comment (obelitz) [inline react/week3/react_week3/src/FetchedData.js:107]: I'd call this component `TodoList`, because that's what it renders

**Review by obelitz — COMMENTED** [review]
What program are you using to write your code?
I'd consider getting a formatter (eg. [Prettier](https://prettier.io/)) which will automatically format the code to make it more readable

File: react/week3/react_week3/src/FetchedData.js (hunk 2 above)
Comment (soujanyapolavarapu) [reply react/week3/react_week3/src/FetchedData.js:156]: I will modify.Thankyou

File: react/week3/react_week3/src/FetchedData.js (hunk 3 above)
Comment (soujanyapolavarapu) [reply react/week3/react_week3/src/FetchedData.js:107]: ok 👍  i understand now :)

Comment (soujanyapolavarapu) [reply react/week3/react_week3/src/FetchedData.js:107]: Thankyou for your valuable time  for reviewing my code and i will definitely update soon according to your comments. :)

Comment (soujanyapolavarapu) [reply react/week3/react_week3/src/FetchedData.js:107]: Hi obelitz, I have updated my code according to your comments.Please confirm me that can i merge now? :) Thank you

**Pull request merged**